### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ccombe**
added my items from the issue #2 including fixing some of my typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,20 @@ Use the checklist below to ask questions about the candidate domain / service /
...
 * **Risk**: what is the cost of risk of splitting this capability or service? Could something go wrong?
 
+### Additional Considerations
+
+* **Skills Liquidity**: Team vs members of the team - considering skills liquidity (Chris Matts)
+* **Loose Couple**: Does this 'thing' make sense to independently migrate / deploy in the public cloud independant of related 'things' 
+* **Tight Coupling**: Do you have any dependencies on vendor / 3rd party software that prevents scaling if demand increases 
+* **Reuse**: Does an existing 'thing' / service already exist on public cloud that you could use as a commoditized service instead 
+* **Commercial Opportunity**: Is there demand for this 'thing' outside of the context of its current usage, could this be used more broadly within your organization or to different customer segments.
+* **Data Govoernance**: Does this 'thing' as as a master or authoritative source for key static / reference / client data?
```
Comment (conjurer-rich) [inline README.md]: Typo 'Governance"

File: README.md [hunk 2]
```diff
@@ -50,6 +50,20 @@ Use the checklist below to ask questions about the candidate domain / service /
...
+* **Interfact Contracts**: Do you have versioned interface contracts and the ability to deploy new versions without impacting existing 'customers' (consumers)
+* **Resilience / scalability**: as demand for your 'thing' scales do you have a linear increase in demand for new capacity / availability (including geographical regions)
+* **Anti-pattern**: Are there since points of failure / individuals that cause a bottleneck
```
Comment (conjurer-rich) [inline README.md]: What does 'Are there **since** points of failure' mean?

File: README.md [hunk 3]
```diff
@@ -50,6 +50,20 @@ Use the checklist below to ask questions about the candidate domain / service /
 * **Wardley Maps**: could the capability or service be outsourced to a SaaS Product or Commodity provider? Will it likely be outsource-able soon? If so, it's a candidate for splitting off as a separate service in preparation for possible outsourcing to a Product or Commodity provider. 
 * **Risk**: what is the cost of risk of splitting this capability or service? Could something go wrong?
 
+### Additional Considerations
+
+* **Skills Liquidity**: Team vs members of the team - considering skills liquidity (Chris Matts)
```
Comment (conjurer-rich) [inline README.md]: What does the reference to (Chris Matts) mean here?

File: README.md [hunk 4]
```diff
@@ -50,6 +50,20 @@ Use the checklist below to ask questions about the candidate domain / service /
...
+* **Loose Couple**: Does this 'thing' make sense to independently migrate / deploy in the public cloud independant of related 'things'
```
Comment (conjurer-rich) [inline README.md]: Typo 'independant' should be 'independent'?

File: README.md [hunk 5]
```diff
@@ -50,6 +50,20 @@ Use the checklist below to ask questions about the candidate domain / service /
 * **Wardley Maps**: could the capability or service be outsourced to a SaaS Product or Commodity provider? Will it likely be outsource-able soon? If so, it's a candidate for splitting off as a separate service in preparation for possible outsourcing to a Product or Commodity provider. 
 * **Risk**: what is the cost of risk of splitting this capability or service? Could something go wrong?
 
+### Additional Considerations
+
+* **Skills Liquidity**: Team vs members of the team - considering skills liquidity (Chris Matts)
+* **Loose Couple**: Does this 'thing' make sense to independently migrate / deploy in the public cloud independant of related 'things' 
+* **Tight Coupling**: Do you have any dependencies on vendor / 3rd party software that prevents scaling if demand increases 
+* **Reuse**: Does an existing 'thing' / service already exist on public cloud that you could use as a commoditized service instead
```
Comment (conjurer-rich) [inline README.md]: Is this repetition of the Wardley Maps point above?

File: README.md (hunk 1 above)
Comment (ccombe) [reply README.md]: thanks - fixing

File: README.md (hunk 2 above)
Comment (ccombe) [reply README.md]: removing

File: README.md (hunk 4 above)
Comment (ccombe) [reply README.md]: fixing

File: README.md (hunk 5 above)
Comment (ccombe) [reply README.md]: yes good point, I had raised these ideas well before the DDD talk so will remove

**Pull request merged**